webcrypto-shim.js

```diff
@@ -539,6 +539,10 @@
                     a.tag = (c.buffer || c).slice( c.byteLength - tl );
                 }
 
+                if ( isIE && ka.name === 'AES-GCM' && !args[0].tagLength ) {
```

Comment:
If `tagLength` of `0` is valid, might want to be more explicit with the condition here:

```suggestion
                if ( isIE && ka.name === 'AES-GCM' && args[0].tagLength === undefined ) {
```